head_test.go

```diff
@@ -275,6 +278,7 @@ func TestHeadDeleteSimple(t *testing.T) {
 
 	head, err := NewHead(nil, nil, nil, 1000)
 	require.NoError(t, err)
+	defer h.Close()
```

Comment:
tests are failing as this needs to be `head.Close()`

Reply:
Whoops copy+paste, must've done something wrong running tests. Will fix when home.